src/core/namespace.c

```diff
@@ -976,7 +976,20 @@ static int clone_device_node(const char *node, const char *temporary_mount, bool
         return 0;
 }
 
-static char *settle_runtime_dir(RuntimeScope scope) {
+static int bind_mount_device_dir(const char *dir, const char *temporary_mount) {
+        const char *t;
+
+        assert(dir);
+        assert(path_is_absolute(dir));
+        assert(temporary_mount);
+
+        t = strjoina(temporary_mount, dir);
```

Comment:
`prefix_roota()`? Then, we can drop `path_is_absolute(dir)`.

Reply:
I'd like to ensure that the caller always passes absolute path. Also this is used below as bind mount src, so relative dir would be confusing?

Reply:
OK. Feel free to ignore.